test/integration/config/esm-good/index.js

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* eslint-disable */
```

Comment:
i get this is just testing but this is a very large hammer.  typically we would use the 

```
/* eslint-disable-next-line <list rules in violation> */
```